### ISSUE TITLE
Initialiazation for large unitcell

### ISSUE THREAD

**Issue by ionuorah** [positivemuon/aiida-impuritysupercellconv#3]
[GP: Question. If the unit cell is already almost
cubic and very large, this will always try to make a super-
cell. Is this wanted? In that case, we should be OK with
a 1 × 1 × 1? IFE: At the moment it starts with creating a
larger cell. We can include some lines to make it skip this
step if the input unitcell is large enough, but then we will
need to define a large lattice parameter threshold for this,
or maybe there is another way. But also, I guess its rare
to find a cubic unitcell with all lattice parameters greater
than 8 ˚A (this value was determined from convergence
test performed in a number of ferromagnetic metals, PRB
97, 174414, 2018) ]

Consider this, [min(l_at + 1, 8), N_at]

So basically if, minimum lattice vector is > 8, do not generate supercell, only generate when less.

Also, check in Nat +1, give same results as Nat
